rfcs/2021-05-23-groups2.md

```diff
@@ -0,0 +1,134 @@
iBHOmdpZEIgU0VOVCBpbnRNc2dJRFxuXG4gIEFBIC0-PiBBOiBHOmdpZEEgTVNHIEM6aWRBQiBpbnRNc2dJRCBtc2dkYXRhXG4gIEEgLT4-IEFBOiBHOmdpZEEgQUNLIGludE1zZ0lEXG4gIEFBIC0-PiBCQTogdmlhIGdpZEFCOiBSQ1ZEIGV4dE1zZ0lEIGhhc2ggc2lnXG4gIEJBIC0-PiBCOiBHOmdpZEEgUkNWRCBDOmlkQkEgaW50TXNnSUQgc3RhdHVzPGJyPihzdGF0dXMgLSBtZXNzYWdlIGludGVncml0eSBjaGVjaylcblxuICBNQSAtPj4gTTogRzpnaWRNIE1TRyBDOmlkTUIgaW50TXNnSUQgbXNnZGF0YVxuICBNIC0-PiBNQTogRzpnaWRNIEFDSyBpbnRNc2dJRFxuICBNQSAtPj4gQkE6IHZpYSBnaWRNQjogUkNWRCBleHRNc2dJRCBoYXNoIHNpZ1xuICBCQSAtPj4gQjogRzpnaWRNIFJDVkQgQzppZEJNIGludE1zZ0lEIHN0YXR1c1xuXG4gIG5vdGUgb3ZlciBCQSwgQjogb25jZSByZWNlaXZlZCBieSBhbGxcbiAgQkEgLT4-IEI6IEc6Z2lkTSBSQ1ZEIGludE1zZ0lEIHN0YXR1c1xuXG4gIG5vdGUgb3ZlciBNLCBCOiA0YS4gQSBsZWF2ZXMgZ3JvdXBcblxuICBBIC0-PiBBQTogRzpnaWRBIExFQVZFXG4gIEFBIC0-PiBBOiBHOmdpZEEgT0tcbiAgQUEgLT4-IEJBOiB2aWEgZ2lkQUI6IExFRlRcbiAgbm90ZSBvdmVyIEFBOiByZW1vdmUgZ2lkQUIsIHJlbW92ZSBmcm9tIGJyb2FkY2FzdFxuICBub3RlIG92ZXIgQkE6IHJlbW92ZSBnaWRCQSwgcmVtb3ZlIGZyb20gYnJvYWRjYXN0XG4gIEJBIC0-PiBCOiBHOmdpZEIgTEVGVCBDOmlkQkFcblxuICBBQSAtPj4gTUE6IHZpYSBnaWRBTTogTEVGVFxuICBub3RlIG92ZXIgQUE6IHJlbW92ZSBnaWRBTSwgcmVtb3ZlIGZyb20gYnJvYWRjYXN0XG4gIG5vdGUgb3ZlciBNQTogcmVtb3ZlIGdpZE1BLCByZW1vdmUgZnJvbSBicm9hZGNhc3RcbiAgTUEgLT4-IE06IEc6Z2lkTSBMRUZUIEM6aWRNQVxuXG4gIEFBIC0-PiBBOiBHOmdpZEEgTEVGVFxuXG4gIG5vdGUgb3ZlciBCLCBCQTogaWYgYWxsIG1lbWJlcnMgbGVmdFxuICBCQSAtPj4gQjogRzpnaWRCOiBFTVBUWVxuXG4gIG5vdGUgb3ZlciBNLCBCOiA0Yi4gQSByZW1vdmVzIEIgZnJvbSBncm91cFxuXG4gIEEgLT4-IEFBOiBHOmdpZEEgUkVNIEM6aWRBQlxuICBBQSAtPj4gQTogRzpnaWRBIE9LXG4gIEFBIC0-PiBCQTogdmlhIGdpZEFCOiBPVVRcbiAgbm90ZSBvdmVyIEJBOiByZW1vdmUgZ2lkQkEsIGFsbCBnaWRCTVxuICBCQSAtPj4gQjogRzpnaWRCIE9VVCBDOmlkQkFcblxuICBub3RlIG92ZXIgQUE6IHJlbW92ZSBnaWRBQlxuICBBQSAtPj4gQTogRzpnaWRBIE9LXG5cbiAgbm90ZSBvdmVyIE0sIEI6IGJlbG93IHN0ZXBzIGhhcHBlbiBmb3IgZWFjaCBleGlzdGluZyBtZW1iZXIgTVxuXG4gIEFBIC0-PiBNQTogdmlhIGdpZEFNOiBSRU0gQzptaWRCXG4gIG5vdGUgb3ZlciBNQTogcmVtb3ZlIGdpZE1CXG4gIE1BIC0-PiBBQTogdmlhIGdpZE1BOiBSRU1EIEM6bWlkQlxuICBNQSAtPj4gTTogRzpnaWRNIFJFTUQgQzppZE1CIEM6aWRNQTxicj4oQiByZW1vdmVkIGJ5IEEpXG5cbiAgbm90ZSBvdmVyIEEsIEFBOiBvbmNlIGFsbCBtZW1iZXJzIHJlbW92ZWQgQlxuXG4gIEFBIC0-PiBBOiBHOmdpZEEgUkVNRCBDOmlkQUI8YnI-KEIgcmVtb3ZlZCBieSB0aGlzIGFnZW50KVxuXG4gIG5vdGUgb3ZlciBNLCBCOiA0Yy4gQSBkZWxldGVzIGdyb3VwXG4gIEEgLT4-IEFBOiBHOmdpZEEgREVMXG4gIEFBIC0-PiBBOiBHOmdpZEEgT0tcbiAgXG4gIEFBIC0-PiBCQTogdmlhIGdpZEFCOiBERUxcbiAgbm90ZSBvdmVyIEJBOiByZW1vdmUgYWxsIGdyb3VwIGNvbm5lY3Rpb25zIGFuZCBtZXNzYWdlc1xuICBCQSAtPj4gQjogRzpnaWRCIERFTEQgQzppZEJBPGJyPihncm91cCBkZWxldGVkIGJ5IEEpXG4gIEJBIC0-PiBBQTogdmlhIGdpZEJBOiBERUxEXG4gIEFBIC0-PiBBOiBHOmdpZEEgREVMRCBDOmlkQUJcblxuICBBQSAtPj4gTUE6IHZpYSBnaWRBTTogREVMXG4gIG5vdGUgb3ZlciBNQTogcmVtb3ZlIGFsbCBncm91cCBjb25uZWN0aW9ucyBhbmQgbWVzc2FnZXNcbiAgTUEgLT4-IE06IEc6Z2lkTSBERUxEIEM6aWRNQTxicj4oZ3JvdXAgZGVsZXRlZCBieSBBKVxuICBNQSAtPj4gQUE6IHZpYSBnaWRNQTogREVMRFxuICBBQSAtPj4gQTogRzpnaWRBIERFTEQgQzppZEFNXG5cbiAgQUEgLT4-IEE6IEc6Z2lkQSBERUxEPGJyPihncm91cCBkZWxldGVkIGJ5IHRoaXMgYWdlbnQgLSBhbGwgY29uZmlybWVkKVxuIiwibWVybWFpZCI6e30sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)
```

Comment:
this link appears to be broken. also does it duplicate the first one?

Reply:
links are fixed now - they are to the same diagram (just one HTML, another svg)